src/main/java/org/terasology/segmentedpaths/events/OnExitSegment.java

```diff
@@ -18,14 +18,27 @@
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.Event;
 
+/**
+ * Event called when an entity which has {@link org.terasology.segmentedpaths.components.PathFollowerComponent} leaves a segment.
+ */
 public class OnExitSegment implements Event {
 
+    /**
+     * EntityRef to an entity, which leaves a segment.
+     */
```

Comment:
Same as last time - `//` instead of `/** */`
Also, `EntityRef to an entity`
As mentioned before, this should be a normal comment, since you are not documenting a method. So, what's the point of mentioning that it's an entityRef, when the programmer can just look at the line below and see that? It's like doing this:
```
// Declare a variable named x, that is of type int
int x;
```
Convert to comment, and shorten it down.

Reply:
On second thought, we don't even need this comment. The variable name is descriptive enough. Scrap this comment.